test/object-omit/index.js

```diff
@@ -1,3 +1,4 @@
+// @ts-check
```

Comment:
The addition of this comment shows TypeScript errors in this file if it doesn't work with the provided type definitions; thereby, validating that the provided type definitions were written accurately.